.github/workflows/build.yml

```diff
@@ -0,0 +1,142 @@
+                    cp etc/travis/sylius18/sylius_api.yaml tests/Application/config/routes/
+                    rm tests/Application/src/Migrations/Version20200604080033.php
+
+            -
+                name: Install Composer dependencies
+                run: composer update --no-progress --prefer-dist --no-interaction
+
+            -
+                name: Composer link templates
+                run: composer link-templates
+
+            -
+                name: Composer validate
+                run: composer validate --strict
+
+            -
+                name: Composer analyse
+                run: composer analyse
+
+            -
+                name: Run phpspec
+                run: vendor/bin/phpspec run
+
+            -
+                name: Run phpunit
+                run: vendor/bin/phpunit
+
+            -
+                name: Setup database, assets and cache
+                run: |
+                    (cd tests/Application && bin/console doctrine:database:create -vvv)
+                    (cd tests/Application && bin/console doctrine:schema:create -vvv)
+                    (cd tests/Application && bin/console assets:install public -vvv)
+                    (cd tests/Application && bin/console cache:warmup -vvv)
+
+            -
+                name: Run behat without javascript
+                run: vendor/bin/behat --strict -vvv --no-interaction --tags="~@javascript" || vendor/bin/behat --strict -vvv --no-interaction --tags="~@javascript" --rerun
+
+            -
+                name: Install and build Yarn dependencies
+                run: |
+                    (cd tests/Application && yarn install)
+                    (cd tests/Application && yarn build)
+
+            -
+                name: Output PHP version
+                run: php -v | head -n 1 | awk '{ print $2 }' > .php-version
+
+            -
+                name: Install certificates
+                run: symfony server:ca:install
+
+            -
+                name: Run Chrome headless
+                run: |
+                    google-chrome-stable --enable-automation --disable-background-networking --no-default-browser-check --no-first-run --disable-popup-blocking --disable-default-apps --allow-insecure-localhost --disable-translate --disable-extensions --no-sandbox --enable-features=Metal --headless --remote-debugging-port=9222 --window-size=2880,1800 --proxy-server='direct://' --proxy-bypass-list='*' http://127.0.0.1 > /dev/null 2>&1 &                    sleep 15
+
+            -
+                name: Run webserver
+                run: (cd tests/Application && symfony server:start --port=8080 --dir=public --daemon)
+
+            -
+                name: Run behat with javascript
+                run: vendor/bin/behat --colors --strict -vvv --no-interaction --tags="@javascript" || vendor/bin/behat --strict -vvv --no-interaction --tags="@javascript" --rerun
```

Comment:
Let's move it just after installing dependencies so that it will give it enough time to boot correctly (for both webserver and headless chrome).

Reply:
Except for run Behat.

Reply:
Could be even moved before installing dependencies.